tests/liteclient_test.cc

```diff
@@ -340,14 +329,22 @@ class LiteClientTest : public ::testing::Test {
         // make sure that the new Target has been applied
         ASSERT_EQ(client.getCurrent().sha256Hash(), to_target.sha256Hash());
         ASSERT_EQ(client.getCurrent().filename(), to_target.filename());
+        // emulate the daemon_main work, this is all looks really ugly
+        // but without pulling the update simplifaction PR changes it's really tough to
+        // make it nice, clean and avoiding code duplication
+        // line 264 in main.cc
```

Comment:
"line 264 in main.cc" isn't going to be valid for long. I'd either drop the comment or maybe add a "TODO code duplication ...." with a versioned reference to main.cc.

Otherwise - this is a nice test. We've had at least 2 regressions on these headers in the past.

Reply:
Updated with "TODO: "